velox/functions/prestosql/tests/MapFromEntriesTest.cpp

```diff
@@ -293,9 +293,6 @@ TEST_F(MapFromEntriesTest, arrayOfDictionaryRowOfNulls) {
     rowVector->setNull(i, true);
   }
 
```

Comment:
ah no for this test we want the child to be of size 0, can you downsize the children insteasd
rowVector->childAt(0)->resize(0)
and keep the checks